vMenu/EventManager.cs

```diff
@@ -34,6 +34,7 @@ public EventManager()
         {
             // Add event handlers.
             EventHandlers.Add("vMenu:SetAddons", new Action(SetAddons));
+            EventHandlers.Add("vMenu:SetExtras", new Action(SetExtras));
```

Comment:
Since these eventhandlers (in theory) could be handled in any random order, I think it'd be safer to combine the extras with the Addons.

So we trigger the `vMenu:SetAddons` event from the server, and on the client, that function calls the `SetExtras` function at the very end, just before it sets `MainMenu.ConfigOptionsSetupComplete = true;`. That way we don't need a new event and we are sure that `MainMenu.ConfigOptionsSetupComplete = true;` only gets set when both addons and extra's have been setup.

Reply:
As suggested, converted into one event handler named SetConfigOptions. Kept the SetAddons event handler for backwards-compatibility in case other mods are using or depend on that event handler, although it also runs the SetExtras function.